projects/angular-auth-oidc-client/src/lib/callback/refresh-session.service.spec.ts

```diff
@@ -1,5 +1,6 @@
 import { fakeAsync, TestBed, tick, waitForAsync } from '@angular/core/testing';
-import { delay, of, throwError } from 'rxjs';
+import { of, throwError } from 'rxjs';
+import { delay } from 'rxjs/operators';
```

Comment:
It seems like adding this import also re-formatted the file.
If needed I can revert this.

Reply:
tip top, thanks